<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsTabWidget</class>
 <widget class="QTabWidget" name="SettingsTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tabDatabases">
   <attribute name="title">
    <string>Databases</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="gridLayoutWidget">
          <layout class="QGridLayout" name="gridLayoutDatabases">
           <item row="0" column="0" colspan="3">
            <widget class="QTreeWidget" name="databasesTreeWidget">
             <property name="minimumSize">
              <size>
               <width>410</width>
               <height>0</height>
              </size>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Host</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Port</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>User</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QToolButton" name="buttonDelete">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="buttonAdd">
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBoxDatabase">
          <property name="title">
           <string>Database</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="5" column="1">
            <widget class="QSpinBox" name="spinBoxConnectTimeout">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>60</number>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="labelRequireSSL">
             <property name="text">
              <string>Require SSL</string>
             </property>
             <property name="buddy">
              <cstring>checkBoxRequireSSL</cstring>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QPlainTextEdit" name="plainTextEditService"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditHost"/>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="labelOptions">
             <property name="text">
              <string>Options</string>
             </property>
             <property name="buddy">
              <cstring>plainTextEditOptions</cstring>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="labelConnectTimeout">
             <property name="text">
              <string>Connect Timeout</string>
             </property>
             <property name="buddy">
              <cstring>spinBoxConnectTimeout</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelUser">
             <property name="text">
              <string>User</string>
             </property>
             <property name="buddy">
              <cstring>lineEditUser</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelName">
             <property name="text">
              <string>Name</string>
             </property>
             <property name="buddy">
              <cstring>lineEditName</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEditPort"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelHost">
             <property name="text">
              <string>Host</string>
             </property>
             <property name="buddy">
              <cstring>lineEditHost</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditName"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="labelPass">
             <property name="text">
              <string>Pass</string>
             </property>
             <property name="buddy">
              <cstring>lineEditPass</cstring>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEditPass">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEditUser"/>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="labelService">
             <property name="text">
              <string>Service</string>
             </property>
             <property name="buddy">
              <cstring>plainTextEditService</cstring>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QCheckBox" name="checkBoxRequireSSL">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelPort">
             <property name="text">
              <string>Port</string>
             </property>
             <property name="buddy">
              <cstring>lineEditPort</cstring>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QPlainTextEdit" name="plainTextEditOptions"/>
           </item>
           <item row="9" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QToolButton" name="buttonTest">
               <property name="text">
                <string>Test settings</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelTestOutput">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>databasesTreeWidget</tabstop>
  <tabstop>buttonDelete</tabstop>
  <tabstop>buttonAdd</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditHost</tabstop>
  <tabstop>lineEditPort</tabstop>
  <tabstop>lineEditUser</tabstop>
  <tabstop>lineEditPass</tabstop>
  <tabstop>spinBoxConnectTimeout</tabstop>
  <tabstop>checkBoxRequireSSL</tabstop>
  <tabstop>plainTextEditOptions</tabstop>
  <tabstop>plainTextEditService</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonTest</sender>
   <signal>clicked()</signal>
   <receiver>labelTestOutput</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
